### DISCUSSION TITLE
How can I change qBittorrent-nox with qBittorrent-nox Enhanced?

### DISCUSSION THREAD

**Discussion by erickyun** [anasty17/mirror-leech-telegram-bot#1356] [Q&A]
Hi,

I have a 1GB DDR4 RAM Ubuntu 20.04 VPS.
I would like to change qBittorrent-nox with qBittorrent-nox Enhanced.
How can I do this? Or can I do this, would this break something? 
I would appreciate if someone could help me please?

**Comment by anasty17** [comment]
Uninstall qbittorrent-nox and installed the enhanced version and do some testes. I didn't try it before

**Reply by erickyun** [reply]
But where does docker install it? I did use `sudo docker-compose up --build` to build the app. And İt seems the package is not in `/usr/bin` ? Would `apt-get uninstall qbittorrent-nox` work?

Could you give me the cmd please sir?

**Reply by anasty17** [reply]
Read more about docker
